stack/core/src/main/java/org/apache/usergrid/persistence/EntityManagerFactory.java

```diff
@@ -189,6 +188,12 @@ public UUID initializeApplication(
 
     public Health getIndexHealth();
 
+    void rebuildAllIndexes(ProgressObserver progressObserver);
```

Comment:
I could be wrong, but I thought I already removed these today in the 2.0-dev branch. Can you pull and re-merge?  I'm going to reference the indexService directly in our rest tier, rather than continue to funnel data through the emf interfaces.  Eventually we'll have more events in the refactored system.